### PR TITLE
Update Magento_BundleGraphQl ReadMe

### PR DESCRIPTION
**Pull request by korovitskyi**
### Description (*)

Update README.md file at Magento_BundleGraphQl module.

### Fixed Issues (if relevant)

1. magento/devdocs magento/devdocs#4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/BundleGraphQl/README.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
 # BundleGraphQl
 
-**BundleGraphQl** provides type and resolver information for the GraphQl module
+**Magento_BundleGraphQl** module extends **Magento_GraphQl** and **Magento_Bundle** modules by provides type and resolver information for the GraphQl module
```
Comment (rogyar) [inline app/code/Magento/BundleGraphQl/README.md]:
```suggestion
**Magento_BundleGraphQl** module extends **Magento_GraphQl** and **Magento_Bundle** modules. The module provides type and resolver information for GraphQL API
```

File: app/code/Magento/BundleGraphQl/README.md [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
 to generate bundle product information.
+
+## Installation details
+
+Before disabling or uninstalling this module, note that the following modules depends on this module:
```
Comment (rogyar) [inline app/code/Magento/BundleGraphQl/README.md]:
```suggestion
Before disabling or uninstalling this module, note that the following modules depend on this module:
```

File: app/code/Magento/BundleGraphQl/README.md [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+
+- Magento_GraphQl
+- Magento_Bundle
+- Magento_QuoteGraphQl
+- Magento_CatalogGraphQl
+- Magento_Store
+
+For information about module installation in Magento 2, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.html).
```
Comment (rogyar) [inline app/code/Magento/BundleGraphQl/README.md]:
```suggestion
For information about enabling or disabling a module in Magento 2, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.html).
```

**Review by rogyar — CHANGES_REQUESTED** [review]
Hi @korovitskyi. Please, check my suggestions below.

File: app/code/Magento/BundleGraphQl/README.md (hunk 3 above)
Comment (korovitskyi) [reply app/code/Magento/BundleGraphQl/README.md]: Hi, @rogyar !
Thank you for review..
Changes was added.

**Comment by magento-engcom-team** [comment]
Hi @korovitskyi
Thank you for your contribution. This pull request has been combined (merged) with https://github.com/magento/magento2/pull/31304 to optimize pull requests queue on magento/magento2 repository
Closing this pull request.

**Comment by magento-engcom-team** [comment]
Hi @korovitskyi
Thank you for your contribution. This pull request has been combined (merged) with https://github.com/magento/magento2/pull/31304 to optimize pull requests queue on magento/magento2 repository
Closing this pull request.

**Comment by magento-engcom-team** [comment]
Hi @korovitskyi
Thank you for your contribution. This pull request has been combined (merged) with https://github.com/magento/magento2/pull/31304 to optimize pull requests queue on magento/magento2 repository
Closing this pull request.

**Comment by sidolov** [comment]
@magento combine 31304

**Comment by magento-engcom-team** [comment]
Hi @sidolov. Thank you for your request. I'm working on combining the pull requests for you

**Pull request closed without merging**